cartographer/common/time.cc

```diff
@@ -50,5 +55,12 @@ common::Duration FromMilliseconds(const int64 milliseconds) {
       std::chrono::milliseconds(milliseconds));
 }
 
+double GetThreadCpuTimeSeconds() {
+  struct timespec thread_cpu_time;
+  CHECK(clock_gettime(CLOCK_THREAD_CPUTIME_ID, &thread_cpu_time) == 0)
+      << "clock_gettime failed: " << std::strerror(errno);
```

Comment:
`CHECK` should already print the precise call that failed, does it not? So the first part `"clock_gettime failed: "` is redundant. Remove?

Reply:
Good point. Done.